incubator/hnc/pkg/kubectl/config_describe.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Go filenames usually don't use underscores except for meaningful suffixes like `_test`. So this should be `configdescribe.go`.

Reply:
Done. Thanks for the suggestion!